share/lib/python/neuron/__init__.py

```diff
@@ -233,10 +233,19 @@ def test_rxd(exitOnError=True):
 # using the idiom self.basemethod = self.baseattr('methodname')
 # ------------------------------------------------------------------------------
 
+import sys, types
+
+# Load the `hclass` factory for the correct Python version 2/3 and prevent the
+# incorrect module source code from being opened by creating an empty module.
```

Comment:
nice. I was unaware of this and hence the str contents of the  hclass....py to avoid a syntax error.

Reply:
Yep, it's even endorsed by the Python devs to spoof modules like this, and they intentionally left the `sys.modules`  lookup in the import machinery to facilitate this practice.